### DISCUSSION TITLE
Stats API: Get all pages with their "All time" stats?

### DISCUSSION THREAD

**Discussion by LekoArts** [plausible/analytics#2080] [Anything Else]
Hi!

I'm looking at https://plausible.io/docs/stats-api and especially its "Time periods" section. I don't see a mention of "All time" that I can access on the Plausible dashboard.

What I want to do:

Get a list of all tracked pages and their respective "All time" stats. How should I model the query against the Stats API for that?

Thanks!

**Comment by RobertJoonas** [comment]
Hi @LekoArts!

A direct "all time" period option is not supported in Stats API unfortunately. Not as convenient, but it's possible to use this with a fixed start date (from when you started collecting stats) and today's date like this:

```
?period=custom&date=2021-01-01,2022-08-03
```

Hope this helps!

**Reply by RobertJoonas** [reply]
I added the `all_time` option idea here as a feature request: https://github.com/plausible/analytics/discussions/2081. Unfortunately I can't make any timeline promises on this, as we're a small team with limited resources and need to take it one step at a time. Thanks for understanding!

**Reply by LekoArts** [reply]
Thanks for the quick answer! Appreciate it.

It's not urgent so I'll use the query you suggested :)
